src/mbgl/gl/context.cpp

```diff
@@ -250,6 +250,7 @@ UniqueRenderbuffer Context::createRenderbuffer(const RenderbufferType type, cons
     bindRenderbuffer = renderbuffer;
     MBGL_CHECK_ERROR(
         glRenderbufferStorage(GL_RENDERBUFFER, static_cast<GLenum>(type), size.width, size.height));
+    bindRenderbuffer = 0;
```

Comment:
🤕 I think the underlying issue is that Framebuffers are "binding objects", meaning that you first bind a framebuffer, then bind other objects *to* the framebuffer (like textures and renderbuffer), similar to how vertex arrays work. What's happening is that once we bind/unbind a framebuffer, we don't restore the values to the renderbuffer/texture state values. This means that our implementation *thinks* that something is bound, while in reality it's not, because the overarching state object (framebuffer) has been unbound.